apps/demo/src/locales/en/ui.json

```
@@ -11,14 +11,19 @@
   "responsiveMenu": "Responsive menu",
   "table": {
     "title": {
+      "tanstackTable": "Tanstack table",
       "localDataTable": "Local data table"
     },
     "headers": {
       "userName": "User name",
       "email": "Email",
       "roles": "Roles",
       "age": "Age",
-      "action": "Action"
+      "action": "Action",
+      "givenName": "Given Name",
+      "middleNames": "Middle Names",
```

Follow sentence case:
```suggestion
      "givenName": "Given name",
      "middleNames": "Middle names",
```